src/styles/variables.css

```diff
@@ -0,0 +1,5 @@
+:root {
+  --color-text-main: #060b35;
+  --color-text-secondary: #616070;
+  --color-buttom: #00acdc;
```

Comment:
There is a typo in the variable name, it should be --color-button instead of --color-buttom. 😊